lxd/instance/drivers/driver_lxc.go

```diff
@@ -1325,9 +1325,6 @@ func (d *lxc) IdmappedStorage(path string, fstype string) idmap.IdmapStorageType
 	if idmap.CanIdmapMount(path, fstype) {
 		// Use idmapped mounts.
 		mode = idmap.IdmapStorageIdmapped
-		d.logger.Error("YAS", logger.Ctx{"path": path, "fstype": fstype})
```

Comment:
These lines were added in the earlier commit, can you rebase this so they dont get added in the first place?

Reply:
Whoops, sure.